eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/ReplicationEndpoint.scala

```diff
@@ -248,8 +248,10 @@ class ReplicationEndpoint(
   private[eventuate] val connectors: Set[SourceConnector] =
     connections.map(new SourceConnector(this, _))
 
+  // lazy to make sure concurrently running (created actors) do not access null-reference
   private[eventuate] lazy val acceptor: ActorRef =
     system.actorOf(Props(new Acceptor(this)), name = Acceptor.Name)
+  acceptor // make sure acceptor is started
```

Comment:
I think we already discussed that in context of a bug but I don't remember the details any more. Do you?